src/dotnet-dev-certs/Program.cs

```diff
@@ -50,6 +51,11 @@ public static int Main(string[] args)
                         "Check for the existence of the certificate but do not perform any action",
                         CommandOptionType.NoValue);
 
+                    var clean = c.Option(
```

Comment:
I'd love to see this separated into its own subcommand. I think we have already opened https://github.com/aspnet/DotNetTools/issues/357 to discuss this.

Reply:
Agreed, and I'd like to hear @danroth27's thought on this as well.